opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipOverview.js

```diff
@@ -281,44 +298,55 @@ class StixSightingRelationshipContainer extends Component {
                   label={t('sighted in/at')}
                 />
               </div>
-              <Link to={`${linkTo}/${to.id}`}>
+              <Link to={!toRestricted ? `${linkTo}/${to.id}` : '#'}>
                 <div
                   className={classes.item}
                   style={{
-                    border: `2px solid ${itemColor(to.entity_type)}`,
+                    border: `2px solid ${itemColor(
+                      !toRestricted ? to.entity_type : 'Unknown',
+                    )}`,
                     top: 20,
                     right: 20,
                   }}
                 >
                   <div
                     className={classes.itemHeader}
                     style={{
-                      borderBottom: `1px solid ${itemColor(to.entity_type)}`,
+                      borderBottom: `1px solid ${itemColor(
+                        !toRestricted ? to.entity_type : 'Unknown',
```

Comment:
why 'Unknown' here instead of 'Restricted' ?

Reply:
it's not a displayed name, it's the color applied in the platform for such restricted entities.